### DISCUSSION TITLE
Applying alpha to theme color values (rgba)

### DISCUSSION THREAD

**Discussion by iffa** [mantinedev/mantine#4882] [Q&A] [answered]
Not strictly a Mantine question, but my CSS-fu is not strong enough for this. Is it possible to apply opacity to theme color values? Something along the likes of this in CSS: `background: rgba(var(--mantine-primary-color-filled), 0.15)`

**Comment by rtivital** [comment, marked as answer]
No, it is not possible, there is no such CSS feature

**Comment by theSdev** [comment]
Seems to be possible using [color-mix](https://gradient.style/#type=linear&space=oklab&linear_named_angle=to+right&linear_angle=90&stops=%7B%22kind%22%3A%22stop%22%2C%22color%22%3A%22color-mix%28in+oklch%2C+oklch%2870%25+0.5+340%29%2C+transparent+50%25%29%22%2C%22auto%22%3A0%2C%22position1%22%3A0%2C%22position2%22%3A0%7D&stops=%7B%22kind%22%3A%22hint%22%2C%22auto%22%3A50%2C%22percentage%22%3A50%7D&stops=%7B%22kind%22%3A%22stop%22%2C%22color%22%3A%22oklch%2890%25+0.5+200%29%22%2C%22auto%22%3A100%2C%22position1%22%3A100%2C%22position2%22%3A100%7D)

Example:
`background-color: color-mix(in srgb, var(--mantine-color-brand-1), transparent 50%);`
